<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialogservoClass</class>
 <widget class="QDialog" name="dialogservoClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>305</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Servo control</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="6" column="1">
      <widget class="QSpinBox" name="spinBoxPos1">
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
       </property>
       <property name="maximum">
        <number>65535</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelDec">
       <property name="text">
        <string>Dec</string>
       </property>
       <property name="buddy">
        <cstring>lineEditDec</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelHex">
       <property name="text">
        <string>Hex</string>
       </property>
       <property name="buddy">
        <cstring>lineEditHex</cstring>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QSpinBox" name="spinBoxPos2">
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
       </property>
       <property name="maximum">
        <number>65535</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelBin">
       <property name="text">
        <string>Bin</string>
       </property>
       <property name="buddy">
        <cstring>lineEditBin</cstring>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="labelPo2">
       <property name="text">
        <string>Servo position 2</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labelSeqLength">
       <property name="text">
        <string>Sequence length</string>
       </property>
       <property name="buddy">
        <cstring>spinBoxSeqLength</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEditHex">
       <property name="text">
        <string>0</string>
       </property>
       <property name="maxLength">
        <number>39</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>manual</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>STI</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SSLOPE</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SCAN_GATED</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>STICNT</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SDAT</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SEC</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>XCK</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BTI</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BSLOPE</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BTICNT</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BDAT</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BEC</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BLOCK_ON</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BLOCK_ON_SYNCED</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="labelTrigSource">
       <property name="text">
        <string>Trigger source</string>
       </property>
       <property name="buddy">
        <cstring>comboBox</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEditBin">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>0000000000000000000000000000000000000000000000000000000000000000</string>
       </property>
       <property name="maxLength">
        <number>128</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="labelPos1">
       <property name="text">
        <string>Servo position 1</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditDec">
       <property name="text">
        <string>0</string>
       </property>
       <property name="maxLength">
        <number>39</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="spinBoxSeqLength">
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>128</number>
       </property>
       <property name="value">
        <number>128</number>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QRadioButton" name="radioButton">
         <property name="text">
          <string>position 1</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_2">
         <property name="text">
          <string>position 2</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Manual position</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Close</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>dialogservoClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>503</x>
     <y>295</y>
    </hint>
    <hint type="destinationlabel">
     <x>442</x>
     <y>330</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
